Core/src/org/sleuthkit/autopsy/coreutils/VideoUtils.java

```diff
@@ -234,6 +239,158 @@ static BufferedImage generateVideoThumbnail(AbstractFile file, int iconSize) {
         }
         return bufferedImage == null ? null : ScalrWrapper.resizeFast(bufferedImage, iconSize);
     }
+    
+    public static boolean canCompressAndScale(AbstractFile file) {
+
+        if (PlatformUtil.getOSName().toLowerCase().startsWith("Windows")) {
```

Comment:
This always returns false because first we convert to lower case and then compare to "Windows" with capital "W"

Reply:
OMG, you are correct.  I need to find where I copied and pasted it from, but it's there too.